src/vit-servicing-station/vit-servicing-station-lib/src/db/mod.rs

```diff
@@ -68,13 +68,22 @@ WHERE
 ";
 
 pub fn load_db_connection_pool(db_url: &str) -> Result<DbConnectionPool, Error> {
+    load_db_connection_pool_with_max_size(db_url, 10)
+}
+
+pub fn load_db_connection_pool_with_max_size(
```

Comment:
What if to combine both these functions into one with the following signature
```
pub fn load_db_connection_pool_with_max_size(db_url: &str, max_size: Option<u32>),
```

Reply:
I'll remove this as it is just a leftover from some experiments I was doing.